benchttp/service.go

```diff
@@ -0,0 +1,8 @@
+package benchttp
+
+// StatsService defines the interface to implement by a
+// service facade inside the application.
+type StatsService interface {
+	// Create stores stats in database.
+	Create() error
+}
```

Comment:
```suggestion
// InsertService defines the interface to implement by a
// data layer facade.
type InsertionService interface {
	Insert() error
}
```

Inside the context of `worker`, this is not a stats service but an insertion service. What `worker` needs is an implementation of something to insert the output of the computation inside a database.

Reply:
I agree, it is clearer :+1: